config/settings/test.py

```diff
@@ -116,3 +116,7 @@
 SKIP_HTML_FILES = '["index.html"]'
 
 HEROKU_APP_NAME = "sfdoc.example.com"
+
+# Make it easy to differentiate between local, staging and prod versions
+ENV_COLOR = env("ENV_COLOR", default=" #1798c1")
```

Comment:
I don't think anything should default to the same color as production.

Reply:
I honestly don't know if the test.py configuration actually even runs as a web app, but sure, it can't hurt to be extra-cautious.

Reply:
536b68e